py-base/environment.yml

```diff
@@ -100,11 +101,28 @@ dependencies:
   # and it may be a good idea to pin it to <2
   - shapely
   - siphon
-  - tensorflow-cpu
+  # - tensorflow-cpu # keras and tensorflow cause our images to become much larger
+  # Which can cause problems with Github Actions running out of storage https://github.com/docker/build-push-action/issues/321
   - unzip
   - windrose
   - xarray
   - xesmf
   - xgcm
   # - xlrd  # deprecated, use openpyxl instead
   - zarr
+  # Tutorial specific, may be dupes to make things explicit
+  - pandas # 23-Myranda
+  - xarray # 23-Myranda
+  - seaborn # 23-Myranda
+  - matplotlib # 23-Myranda
```

Comment:
If we can remove qt from other places this should save some space too.

```suggestion
  - matplotlib-base # 23-Myranda
```

Reply:
Ahh, jupyter is also to blame. jupyterlab depends on jupyter_core, so giving that a shot now.

<img width="455" alt="image" src="https://github.com/oceanhackweek/jupyter-image/assets/1296209/b33b9ecc-96ef-40b6-a020-7a33baacbde6">

Reply:
That looks like it drops things from 5.46 GB to 4.91!